examples/styling/src/main.rs

```diff
@@ -108,6 +108,7 @@ impl Sandbox for Styling {
             column!["Scroll me!", vertical_space(800), "You did it!"]
                 .width(Length::Fill),
         )
+        .width(300)    
```

Comment:
Passing `Length::Fill` here should work as well. Let's do that.

```suggestion
        .width(Length:Fill)    
```

This kind of layout inconsistency will be fixed soon.